runtime/karura/src/lib.rs

```diff
@@ -1165,26 +1165,48 @@ parameter_types! {
 	pub const MaxPending: u16 = 32;
 }
 
-/// The type used to represent the kinds of proxying allowed.
-#[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Encode, Decode, RuntimeDebug, MaxEncodedLen)]
-pub enum ProxyType {
-	Any,
-	CancelProxy,
-}
-impl Default for ProxyType {
-	fn default() -> Self {
-		Self::Any
-	}
-}
 impl InstanceFilter<Call> for ProxyType {
 	fn filter(&self, c: &Call) -> bool {
 		match self {
 			ProxyType::Any => true,
 			ProxyType::CancelProxy => matches!(c, Call::Proxy(pallet_proxy::Call::reject_announcement(..))),
+			ProxyType::Governance => {
+				matches!(
+					c,
+					Call::Authority(..)
+						| Call::Democracy(..) | Call::GeneralCouncil(..)
+						| Call::FinancialCouncil(..)
+						| Call::HomaCouncil(..) | Call::TechnicalCommittee(..)
+						| Call::Treasury(..) | Call::Bounties(..)
+						| Call::Tips(..) | Call::Utility(..)
```

Comment:
I think we can make `Call::Utility(..)` always allowed no matter type?

Reply:
```
fn filter(&self, c: &Call) -> bool {
    match self {
        _ if matches!(c, Call::Utility(..)) => true,
        ProxyType::Any => true,
        ...
```

It doesn't make sense to do. This will only make `Call::Utility(..)` skip the check, but the execution still fails.(e.g. addProxy `ProxyType::Governance` then use `Utility` to transfer. It will fail in `utility.BatchInterrupted`)